cfme/tests/configure/test_visual_infra.py

```diff
@@ -56,160 +50,160 @@ def value(request):
 
 
 @pytest.mark.meta(blockers=[1267148])
-def test_infra_grid_page_per_item(visual, request, page, value, set_grid, appliance):
+def test_infra_grid_page_per_item(appliance, request, page, value, set_grid):
     """ Tests grid items per page
 
     Metadata:
-        test_flag: visuals
+        test_flag: appliance.user.my_settings.visuals
     """
     if isinstance(page, six.string_types):
         page = getattr(appliance.collections, page)
-    if visual.grid_view_limit != value:
-        visual.grid_view_limit = int(value)
+    if appliance.user.my_settings.visual.grid_view_limit != value:
+        appliance.user.my_settings.visual.grid_view_limit = int(value)
     request.addfinalizer(lambda: go_to_grid(page))
-    limit = visual.grid_view_limit
+    limit = appliance.user.my_settings.visual.grid_view_limit
     view = navigate_to(page, 'All', use_resetter=False)
-    view.toolbar.view_selector.select("Grid View")
+    view.toolbar.view_selector.select('Grid View')
     max_item = view.entities.paginator.max_item
     item_amt = view.entities.paginator.items_amount
     if int(item_amt) >= int(limit):
-        assert int(max_item) == int(limit), "Gridview Failed for page {}!".format(page)
+        assert int(max_item) == int(limit), 'Gridview Failed for page {}!'.format(page)
     assert int(max_item) <= int(item_amt)
 
 
 @pytest.mark.meta(blockers=[1267148])
-def test_infra_tile_page_per_item(visual, request, page, value, set_tile, appliance):
+def test_infra_tile_page_per_item(appliance, request, page, value, set_tile):
     """ Tests tile items per page
 
     Metadata:
-        test_flag: visuals
+        test_flag: appliance.user.my_settings.visuals
     """
     if isinstance(page, six.string_types):
         page = getattr(appliance.collections, page)
-    if visual.tile_view_limit != value:
-        visual.tile_view_limit = int(value)
+    if appliance.user.my_settings.visual.tile_view_limit != value:
+        appliance.user.my_settings.visual.tile_view_limit = int(value)
     request.addfinalizer(lambda: go_to_grid(page))
-    limit = visual.tile_view_limit
+    limit = appliance.user.my_settings.visual.tile_view_limit
     view = navigate_to(page, 'All', use_resetter=False)
     view.toolbar.view_selector.select('Tile View')
     max_item = view.entities.paginator.max_item
     item_amt = view.entities.paginator.items_amount
     if int(item_amt) >= int(limit):
-        assert int(max_item) == int(limit), "Tileview Failed for page {}!".format(page)
+        assert int(max_item) == int(limit), 'Tileview Failed for page {}!'.format(page)
     assert int(max_item) <= int(item_amt)
 
 
 @pytest.mark.meta(blockers=[1267148])
-def test_infra_list_page_per_item(visual, request, page, value, set_list, appliance):
+def test_infra_list_page_per_item(appliance, request, page, value, set_list):
     """ Tests list items per page
 
     Metadata:
-        test_flag: visuals
+        test_flag: appliance.user.my_settings.visuals
```

Comment:
I think in these Metadata instance you want to leave the base 'visuals' string, because it ties into the provider configuration in cfme_data.yaml during collection.  It's included in some of the provider's excluded_flags, though this test isn't parametrized against providers.

Reply:
Ah. This was probably a case of `s/visuals/appliance.user.mysettings.visuals/g`